install.sh

```diff
@@ -30,32 +51,37 @@ else
 fi
 
 if [[ "${VERSION:-}" == "" ]]; then
-  RELEASE_URL=https://github.com/mamba-org/micromamba-releases/releases/latest/download/micromamba-$PLATFORM-$ARCH
+  RELEASE_URL="https://github.com/mamba-org/micromamba-releases/releases/latest/download/micromamba-${PLATFORM}-${ARCH}"
 else
-  RELEASE_URL=https://github.com/mamba-org/micromamba-releases/releases/download/micromamba-$VERSION/micromamba-$PLATFORM-$ARCH
+  RELEASE_URL="https://github.com/mamba-org/micromamba-releases/releases/download/micromamba-${VERSION}/micromamba-${PLATFORM}-${ARCH}"
 fi
 
-BIN_FOLDER=~/.local/bin
-mkdir -p $BIN_FOLDER
-curl $RELEASE_URL -o $BIN_FOLDER/micromamba -fsSL --compressed ${CURL_OPTS:-}
-chmod +x $BIN_FOLDER/micromamba
 
-if [ -t 0 ] ; then
-  printf "Init shell? [Y/n] "
-  read YES
-  printf "Prefix location? [~/micromamba] "
-  read PREFIXLOCATION
-else
-  YES="yes"
-fi
+# Downloading artifact
+mkdir -p "${BIN_FOLDER}"
+curl "${RELEASE_URL}" -o "${BIN_FOLDER}/micromamba" -fsSL --compressed ${CURL_OPTS:-}
+chmod +x "${BIN_FOLDER}/micromamba"
+
 
-if [[ "${PREFIXLOCATION:-}" == "" ]]; then
-  PREFIXLOCATION="~/micromamba"
+# Initializing conda-forge
+if [[ "$CONDA_FORGE_YES" == "" || "$CONDA_FORGE_YES" == "y" || "$CONDA_FORGE_YES" == "Y" || "$CONDA_FORGE_YES" == "yes" ]]; then
+  "${BIN_FOLDER}/micromamba" config append channels conda-forge
+  "${BIN_FOLDER}/micromamba" config append channels nodefaults
+  "${BIN_FOLDER}/micromamba" config set channel_priority strict
```

Comment:
Did you check that `micromamba config ...` works correctly? When I checked last time, it didn't work as expected.

Reply:
I tested the script and the `config` commands work as expected. 

> When I checked last time, it didn't work as expected

Then if the user injects `VERSION` environment variable to an old version, this may fail.
It is unlikely but we could swap this with shell initialization so that is is done nonetheless.